packages/kotti-ui/source/kotti-tag/KtTag.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+	<div class="kt-tag">
+		<div class="kt-tag__text" v-text="text" />
+		<div v-if="!isDisabled" class="kt-tag__icon" @click="$emit('close')">
```

Comment:
I'd `click.prevent.stop` here

we already emit close, usually it's better to avoid the hassle of an unnecessary bubble-up